### DISCUSSION TITLE
Git Storage commiter question.

### DISCUSSION THREAD

**Discussion by aoisensi** [requarks/wiki#6832] [Help / Questions] [answered]
When using Git Storage, is it possible to change the e-mail of the author of a commit to a dummy or eliminate the author itself?

**Comment by NGPixel** [comment, marked as answer]
No, the email of the author of the change is used. You would need to modify the git module code directly and hardcode the email you want:

https://github.com/requarks/wiki/blob/main/server/modules/storage/git/storage.js#L311
and all other similar lines in this file.
